fastlane/lib/commands.rb

```diff
@@ -30,7 +30,7 @@ def authorize_ci_for_keys
 	netrc = Netrc.read
 	unless netrc["github.com"]
 		UI.message "An entry for github.com was not found in ~/.netrc; setting..."
-		netrc["github.com"] = token
+		netrc["github.com"] = token, ''
```

Comment:
We have no idea how this was working on Circle - the netrc gem hasn’t updated in 4 years, and is documented to expect a tuple as input here…